kong/runloop/plugins_iterator.lua

```diff
@@ -335,14 +344,18 @@ local function iterate(self, phase, ctx)
   local iteration = {
     configure = MUST_LOAD_CONFIGURATION_IN_PHASES[phase],
     loaded = self.loaded,
-    phases = ws.phases[phase] or EMPTY_T,
+    phases = ws.phases[phase] or {},
     combos = ws.combos,
     map = ws.map,
     ctx = ctx,
     i = 0,
   }
```

Comment:
It seem that in `get_next_no_ctx` scenario we will only need:
```lua
local iteration = {
  loaded = self.loaded,
  phases = ws.phases[phase] or {},
  i = 0,
}
```

Perhaps a worth to create a different (smaller) table in that case?